Traceback (most recent call last):
  File "/Users/gy22808/opt/anaconda3/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/gy22808/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/gy22808/opt/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/gy22808/opt/anaconda3/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/gy22808/opt/anaconda3/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/gy22808/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/gy22808/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/gy22808/opt/anaconda3/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
## use word cloud to visualize the frequency of poi types mentioned in Bristol 
# first you need to install the package of wordcloud 
from wordcloud import WordCloud, STOPWORDS
import matplotlib.pyplot as plt

bristol_pois_doc = " ".join(list(bristol_pois['fclass']))

wordcloud = WordCloud(width = 800, height = 800,
                background_color ='white',
                min_font_size = 10).generate(bristol_pois_doc)

# plot the WordCloud image                      
plt.figure(figsize = (8, 8), facecolor = None)
plt.imshow(wordcloud)
plt.axis("off")
plt.tight_layout(pad = 0)
 
plt.show()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [0;32mIn [11][0m, in [0;36m<cell line: 3>[0;34m()[0m
[1;32m      1[0m [38;5;66;03m## use word cloud to visualize the frequency of poi types mentioned in Bristol [39;00m
[1;32m      2[0m [38;5;66;03m# first you need to install the package of wordcloud [39;00m
[0;32m----> 3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mwordcloud[39;00m [38;5;28;01mimport[39;00m WordCloud, STOPWORDS
[1;32m      4[0m [38;5;28;01mimport[39;00m [38;5;21;01mmatplotlib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpyplot[39;00m [38;5;28;01mas[39;00m [38;5;21;01mplt[39;00m
[1;32m      6[0m bristol_pois_doc [38;5;241m=[39m [38;5;124m"[39m[38;5;124m [39m[38;5;124m"[39m[38;5;241m.[39mjoin([38;5;28mlist[39m(bristol_pois[[38;5;124m'[39m[38;5;124mfclass[39m[38;5;124m'[39m]))

[0;31mModuleNotFoundError[0m: No module named 'wordcloud'
ModuleNotFoundError: No module named 'wordcloud'

